class/defaults.yml

```
@@ -42,6 +43,42 @@ parameters:
                 tags: []
                 userDataSecret:
                   name: worker-user-data
+    cpDefaultSpecs:
```

While I don't mind abbreviations in the implementation, I much prefer writing out the actual thing in the API

```suggestion
    controlPlaneDefaultSpecs:
```